src/rustup.rs

```diff
@@ -60,16 +76,23 @@ pub fn installed_toolchains(msg_info: &mut MessageInfo) -> Result<Vec<String>> {
         .collect())
 }
 
-pub fn available_targets(toolchain: &str, msg_info: &mut MessageInfo) -> Result<AvailableTargets> {
+pub fn available_targets(
+    // this is explicitly a string and not `QualifiedToolchain`, this is because we use this as a way to ensure that
```

Comment:
This is rather a nit but I'm not a fan of extremely long lines in comments, since these are important to read and I think a limit of ~100 characters per line is great. At least, 165 is pretty prohibitive for those who use split-screen in their editor.